src/OAuth2.php

```diff
@@ -1377,12 +1378,12 @@ private function coerceUri($uri)
     /**
      * @param string $idToken
      * @param string|array|null $publicKey
-     * @param array $allowedAlgs
+     * @param string $allowedAlg
      * @return object
      */
-    private function jwtDecode($idToken, $publicKey, $allowedAlgs)
+    private function jwtDecode($idToken, $publicKey, $allowedAlg)
```

Comment:
unfortunately this is a backwards breaking change.

Reply:
I thought that might be the case. Thanks for the feedback and for the work done in #391. Ultimately, I just need the issue fixed 😄